stack-packs/packs/angular.js

```diff
@@ -7,30 +7,14 @@
-  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by minifying their CSS and JS files in the context of the Angular framework. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  unminified_warning: 'If you are using Angular CLI, ensure that builds are generated in production mode. [Learn more](https://angular.io/guide/deployment#enable-runtime-production-mode).',
-  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by removing unused Javascript files in the context of the Angular framework. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  unused_javascript: 'If you are using Angular CLI, include source maps into your production build to inspect your bundles. [Learn more](https://angular.io/guide/deployment#inspect-the-bundles).',
-  /** Additional description of a Lighthouse audit that tells the user how they can use responsive images in the context of the Angular framework. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  uses_responsive_images: 'Consider using the `BreakpointObserver` utility in the Component Dev Kit (CDK) to manage image breakpoints. [Learn more](https://material.angular.io/cdk/layout/overview).',
-  /** Additional description of a Lighthouse audit that tells the user how they can use preload to improve performance in the context of the Angular framework. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
-  uses_rel_preload: 'Preload routes ahead of time to speed up navigation. [Learn more](https://web.dev/route-preloading-in-angular/).',
-  /** Additional description of a Lighthouse audit that tells the user and *how* they should reduce the size of the web page's DOM in the context of the Angular framework. Links in (parenthesis) become link texts to additional documentation. */
-  dom_size: 'Consider virtual scrolling with the Component Dev Kit (CDK) if very large lists are being rendered. [Learn more](https://web.dev/virtualize-lists-with-angular-cdk/).',
-};
-
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
 module.exports = {
   id: 'angular',
-  iconDataURL: angularIcon,
+  iconDataURL: icon,
   title: 'Angular',
   descriptions: {
     'total-byte-weight': str_(UIStrings.total_byte_weight),
```

Comment:
Having all these definitions added here manually might lead to missed values when the NPM package is updated. This problem might occur when a new audit description is added between versions. 

Technically we could loop over all keys of the `UI_Strings` export of the stack-pack. Sadly we do have some outliers like `UIStrings.unminified_warning`, where this string is used for two audits. I do think we keep them as separate strings inside the stack pack. We could dedupe the same strings before sending them to the translators.

Reply:
Could this be populated automatically? Have a map <stack-pack-string-name, audit-id> in the stack pack repo and build the list here dynamically? `'<audit-id>': str_(UIStrings.<stack-pack-string-name>),`

Reply:
Yes we can, I can put it all in the stack-packs repo, wasn't sure we wanted to.